core/src/main/java/org/apache/carbondata/core/indexstore/BlockletDataMapIndexStore.java

```diff
@@ -244,4 +247,22 @@ public void clearAccessCount(
       cacheable.clear();
     }
   }
+
+  public static class BlockMetaInfo {
+    private String[] locationInfo;
+    private long size;
```

Comment:
please add comment for these two variable

Reply:
ok